src/js/pensions/containers/ConfirmationPage.jsx

```diff
@@ -54,8 +55,18 @@ class ConfirmationPage extends React.Component {
               <strong>Date received</strong><br/>
               <span>{moment(form.submission.submittedAt).format('MMM D, YYYY')}</span>
             </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.confirmationNumber}</span>
+            </li>
+            <li>
+              <strong>Your claim was sent to</strong><br/>
+              <address className="schemaform-address-view">{regionalOffice.map((line, index) =>
+                <p key={index}>{line}</p>)}</address>
+            </li>
           </ul>
         </div>
+        <p>Need help? If you have questions, call <a href="tel:800-827-1000">800-827-1000</a> from 8:00 a.m. - 9:00 p.m. EST Mon - Fri and have the Veteran’s Social security number or VA file number ready. For telecommunication relay services, dial <a href="tel:711">711</a>.</p>
```

Comment:
It does [not display on the confirmation page](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/common/schemaform/FormApp.jsx#L128).